package.json

```
@@ -2,23 +2,36 @@
   "name": "level-lazy-open",
   "version": "1.0.2",
   "description": "Lazily open a leveldown compatible backend",
+  "license": "MIT",
   "scripts": {
-    "test": "standard && nyc node test.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
+    "test": "standard && hallmark && nyc node test.js",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "hallmark": "hallmark --fix"
+  },
+  "dependencies": {
+    "abstract-leveldown": "^2.6.1",
+    "debug": "^3.0.0",
+    "deferred-leveldown": "^2.0.0",
+    "inherits": "^2.0.3"
   },
-  "repository": "level/level-lazy-open",
   "devDependencies": {
     "coveralls": "^3.0.2",
+    "hallmark": "^0.1.0",
+    "level-community": "^3.0.0",
     "memdown": "^1.2.4",
     "nyc": "^12.0.2",
     "standard": "^11.0.1",
     "tape": "^4.6.3"
   },
-  "license": "MIT",
-  "dependencies": {
-    "abstract-leveldown": "^2.6.1",
-    "debug": "^3.0.0",
-    "deferred-leveldown": "^2.0.0",
-    "inherits": "^2.0.3"
-  }
+  "hallmark": {
+    "community": "level-community",
+    "ignore": [
+      "README.md"
+    ]
+  },
+  "repository": "level/level-lazy-open",
```

Don't apply yet, will wait for review of automated commits before tweaking things manually.

```suggestion
  "repository": "Level/lazy-open",
```